bears/general/AnnotationBear.py

```diff
@@ -0,0 +1,134 @@
+from coalib.parsing.StringProcessing.Core import (search_for,
+                                                  search_in_between,
+                                                  unescaped_search_in_between)
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.HiddenResult import HiddenResult
+from coalib.results.SourceRange import SourceRange
+from coalib.results.AbsolutePosition import AbsolutePosition
+
+
+class AnnotationBear(LocalBear):
+
+    def run(self, filename, file, language: str, language_family: str):
+        """
+        Finds out all the positions of strings and comments in a file.
+        The Bear searches for valid comments and strings and yields their
+        ranges as SourceRange objects in HiddenResults.
+
+        :param language:        Language to be whose annotations are to be
+                                searched.
+        :param language_family: Language family whose annotations are to be
+                                searched.
+        :return:                Two HiddenResults first containing a tuple of
+                                SourceRanges of strings and the second
+                                containing a tuple of SourceRanges of comments.
+        """
+        lang_dict = LanguageDefinition(language, language_family)
+        print(file)
+        # Strings
+        strings = dict(lang_dict["string_delimiters"])
+        strings.update(lang_dict["multiline_string_delimiters"])
+        strings_found = self.find_with_start_end(
+                                    filename, file, strings, escape=True)
```

Comment:
do I see this right that you treat string delimiters just like multiline string delimiters?